sql/firefox_accounts_exact_mau28_raw_v1.sql

```diff
@@ -0,0 +1,38 @@
+WITH
+  inactive_days AS (
+    SELECT
+      *,
+      DATE_DIFF(submission_date, date_last_seen, DAY) AS _inactive_days,
+      DATE_DIFF(submission_date, date_last_seen_in_tier1_country, DAY) AS _inactive_days_tier1
+    FROM
+      fxa_users_last_seen_v1
+  )
+
+SELECT
+  submission_date,
+  CURRENT_DATETIME() AS generated_time,
+  COUNTIF(_inactive_days < 28) AS mau,
+  COUNTIF(_inactive_days < 7) AS wau,
+  COUNTIF(_inactive_days < 1) AS dau,
+  -- We are generally using an "exclusive dimensions" methodology where only
+  -- the last country observed for a user is considered for determining whether
+  -- they contribute to tier 1 MAU, but we also include an "inclusive" tier 1
+  -- mau calculation that we have previously been using for KPI calculations on
+  -- FxA data; a user is included in this calculation if it appeared in a tier 1
+  -- country during any of the 28 days in the MAU window.
```

Comment:
note: this isn't quite accurate, due to `udf_mode_last`, but explaining it more precisely I think would make it the explanation too complicated. still, if you can think of a simple way to say `appeared in a tier_1 country more than any other country for at least one of the 28 days in the MAU window.` or something I think that would be better.